CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+<a name="2.0.0-rc.13"></a>
+
+# [2.0.0-rc.13](https://github.com/dherges/ng-packagr/compare/v2.0.0-rc.12...v2.0.0-rc-13) (2018-01-27)
+
+### Bug Fixes
+
+* add `postcss-discard-comments` as a dependency ([#544](https://github.com/dherges/ng-packagr/issues/544)) ([bce5705](https://github.com/dherges/ng-packagr/commit/bce5705)), closes [#543](https://github.com/dherges/ng-packagr/issues/543)
+
+### Features
+
+* added custom rollup-plugin-node-resolve feature ([7b3a64d](https://github.com/dherges/ng-packagr/commit/7b3a64d))
+* added custom rollup-plugin-node-resolve feature ([2c208cd](https://github.com/dherges/ng-packagr/commit/2c208cd))
```

Comment:
Hi @jdbranham,

I know you put some of your time into making the PR.

There are two things why I am going to close the PR w/o merging:

1. Passing through "plain" options for a tool goes against the [initial design of ng-packagr](
https://github.com/dherges/ng-packagr/blob/master/docs/DESIGN.md#configuration-and-customization). The tools being used internally may change or the way how they are being used may change. I am midway throgh a changeset that will allow you (and other users, obviously) to customize the build process. You will then be able to override or replace a transformation step. Please be a little patient vor v2 final.

2. A PR should not update the changelog or cut a release.

Nevertheless, I am interested in why you'd like to override the rollup options? What is it you're trying to achieve?

Reply:
I'm not the submitter obviously, but I was looking into a way to add pug support via a rollup plugin (e.g. `rollup-plugin-pug`)

Reply:
thanks @dherges 

I was bundling a 'browser only' package, and needed rollup to prefer my locally installed dependency, rather than the built in NodeJS version.